src/main/java/com/gd/clinic/security/jwt/TokenUtils.java

```diff
@@ -0,0 +1,58 @@
+package com.gd.clinic.security.jwt;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.security.Keys;
+import io.swagger.models.auth.In;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class TokenUtils {
```

Comment:
you can use lombok annotation `@UtilityClass` and remove static modifiers from methods, they will be added automatically